docs/how-to/rt-conf-config.yaml

```
@@ -0,0 +1,53 @@
+# Several fields use the CPU Lists format. 
+# For the CPU Lists spec, refer to
+# https://docs.kernel.org/admin-guide/kernel-parameters.html#cpu-lists
+
+# Kernel command line parameters
+# TODO add reboot requirement
+kernel_cmdline:
+  # Isolate CPUs from general execution
+  # Format: CPU Lists
+  # isolcpus: "2-3"
+
+  # Enable/disable dynamic ticks during idle time
+  # Supported values: on | off
+  # nohz: "on"
+
+  # Enable/disable full dynamic ticks
+  # Format: CPU Lists
+  # nohz_full: "2-3"
+
+  # Allocated CPUs for kernel threads
+  # Format: CPU Lists
+  # kthread_cpus: "0-1"
+
+  # Allocate CPUs for IRQ handling
+  # Format: CPU Lists
+  # irqaffinity: "0-1"
+
+# Runtime options for IRQ affinity
+irq_tuning:
+  # - # CPUs to which the IRQs are to be moved
+  #   # Format: range, e.g. 0-2
+  #   cpus: "2-3"
+  #   # Arguments used to filter IRQs
+  #   filter:
+  #     actions: "iwlwifi"
+  #     chip_name: "IR-PCI"
+  #     name: "edge"
+  #     type: "edge"
+
+# Runtime options for CPU frequency scaling
+cpu_governance: 
+  #  - # CPUs to which the scaling_governor options are to be applied
+  #    # Format: CPU Lists
+  #    cpus: "0-1"
+  #    # CPU frequency scaling governor
+  #    # Supported values depends on the system, typically: performance, powersave
+  #    scaling_governor: "performance"
+  #    # Minimum CPU frequency
+  #    # Format: frequency with unit, one of "GHz", "MHz", "kHz", "Hz"
+  #    min_frequency: "1GHz"
+  #    # Maximum CPU frequency
+  #    # Format: same as min_frequency
+  #    max_frequency: "5.2gHz"
```

We now abbreviate the **frequency** to **freq** on the config file.

```suggestion
  #    min_freq: "1GHz"
  #    # Maximum CPU frequency
  #    # Format: same as min_frequency
  #    max_freq: "5.2gHz"
```